rskj-core/src/main/java/co/rsk/peg/federation/FederationMember.java

```diff
@@ -37,6 +38,10 @@
  */
 public final class FederationMember {
     public static final FederationMemberPubKeysComparator BTC_RSK_MST_PUBKEYS_COMPARATOR = new FederationMemberPubKeysComparator();
+    private static final int FEDERATION_MEMBER_LIST_SIZE = 3;
+    private static final int FEDERATION_MEMBER_BTC_KEY_INDEX = 0;
+    private static final int FEDERATION_MEMBER_RSK_KEY_INDEX = 1;
+    private static final int FEDERATION_MEMBER_MST_KEY_INDEX = 2;
```

Comment:
Do we need the FEDERATION_MEMBER prefix? It's already part of FederationMember class

Reply:
good catch